tests/Application/Commands/DeleteStalePaymentDetailsTest.php

```diff
@@ -51,7 +44,7 @@ public function testDeletingOneStaleCard(): void
 
         $stripeCustomersProphecy = $this->prophesize(CustomerService::class);
         $stripeCustomersProphecy->search([
-            'query' => "created<{$previousDay->getTimestamp()} and metadata['hasPasswordSince']:null",
+            'query' => "created<{$previousDay->getTimestamp()} and metadata['hasPasswordSince']:null and metadata['paymentMethodsCleared']:null",
```

Comment:
If there's a way to write a search query that will only return accounts with attached payment methods it would be simpler and a bit more elegant, but I think this should work fine and is good enough.

Reply:
@bdsl I'm pretty sure theere isn't. https://stripe.com/docs/search#query-fields-for-customers